happy/State.py

```diff
@@ -878,3 +878,17 @@ def getNodeInfo(self, node, state=None, happy_only=False):
                     node_info[i[0]] = extState[node]
 
         return node_info
+
+    def getExtensionValue(self, extension_name, state=None):
+        """
+        extension_name example: weave-fabric-id
+        """
+        ext_look_list = extension_name.split("-")
```

Comment:
What would happen if I call this with 

    happy-state --extension weave

Reply:
> What would happen if I call this with
> 
> ```
> happy-state --extension weave
> ```

jexie@jexie0:~/github/openweave-core/src/test-apps/happy/topologies/standalone$ happy-state --extension weave
happy topology extension weave value: 
{u'node': {u'BorderRouter': {u'eui64': u'f4-6f-cb-ec-72-80-11-e9', u'iid': u'f66f:cbec:7280:11e9', u'pairing_code': u'5FG9UW', u'weave_node_id': u'f46fcbec728011e9'}, u'ThreadNode': {u'eui64': u'f5-c2-5a-32-72-80-11-e9', u'iid': u'f7c2:5a32:7280:11e9', u'pairing_code': u'XELD7T', u'weave_node_id': u'f5c25a32728011e9'}}, u'fabric': {u'global_prefix': u'fd00:0000:fab1::/48', u'id': u'fab1'}, u'network': {}}

Reply:
I will change it to this format:
jexie@jexie0:~/github/happy$ happy-state --extension weave
weave extension state:
{
  "fabric": {
    "global_prefix": "fd00:0000:fab1::/48", 
    "id": "fab1"
  }, 
  "network": {}, 
  "node": {
    "BorderRouter": {
      "eui64": "f4-6f-cb-ec-72-80-11-e9", 
      "iid": "f66f:cbec:7280:11e9", 
      "pairing_code": "5FG9UW", 
      "weave_node_id": "f46fcbec728011e9"
    }, 
    "ThreadNode": {
      "eui64": "f5-c2-5a-32-72-80-11-e9", 
      "iid": "f7c2:5a32:7280:11e9", 
      "pairing_code": "XELD7T", 
      "weave_node_id": "f5c25a32728011e9"
    }
  }
}